gateways/http/account/create.go

```diff
@@ -60,6 +60,7 @@ func (h Handler) CreateAccount(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(201)
```

Comment:
just a small note here: I suggest you always use the constant provided by the `net/http` package to represent the status code, in this case `http.StatusCreated`